### ISSUE TITLE
python train.py

### ISSUE THREAD

**Issue by develooper1994** [fancoo/WaveGAN-pytorch#8]
def batch_generator(audio_path_list, batch_size):
    streamers = []
    for audio_path in audio_path_list:
        s = pescador.Streamer(sample_generator, audio_path)
        streamers.append(s)

    mux = pescador.ShuffledMux(streamers)  # <--------
    batch_gen = pescador.buffer_stream(mux, batch_size)

    return batch_gen

File "/home/selcuk/PycharmProjects/WaveGAN-pytorch/utils.py", line 108, in batch_generator
    mux = pescador.ShuffledMux(streamers)
  File "/home/selcuk/anaconda3/lib/python3.7/site-packages/pescador/mux.py", line 567, in __init__
    raise PescadorError('Cannot mux an empty collection')
